types/fparser/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for fparser 2.0
+// Project: https://github.com/bylexus/fparse
+// Definitions by: bylexus <https://github.com/bylexus>,
+//                 andrew0687 <https://github.com/andrew0687>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default class Formula {
+    constructor(fStr: string, options?: object);
```

Comment:
the options are as simple as:
```suggestion
    constructor(fStr: string, options?: { memoization?: boolean | undefined });
```